AfroLayout.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
   s.name             = "AfroLayout"
-  s.version          = "0.1.6"
+  s.version          = "3.0.0"
```

Comment:
I have used the convention to start numbering at 3.0.0 for all other pods when they have been converted to Swift3.

Updating to 0.2.0 would also work.